perllib/Open311/PopulateServiceList.pm

```diff
@@ -267,6 +267,9 @@ sub _add_meta_to_contact {
         @meta = grep { ! $ignore{ $_->{ code } } } @meta;
     }
 
+    @meta = map { delete $_->{automated}; $_; }
+        grep { !defined $_->{automated} || !$_->{ automated } } @meta;
```

Comment:
This should only ignore `server_set` ones. If they are `hidden_field` it needs to let them be added as normal, and then `category_extra_hidden` can be adjusted to hide those rather than hardcoding a list.